tools/webpack/shared.js

```diff
@@ -48,6 +48,13 @@ const baseConfig = {
 			},
 		],
 	},
+	resolve: {
+		// Ensure "require" has a higher priority when matching export conditions.
+		// Needed for wasm-vips which is used by the @wordpress/vips package.
+		// https://webpack.js.org/configuration/resolve/#resolveconditionnames
+		// https://github.com/kleisauke/wasm-vips/issues/50#issuecomment-1664118137
+		conditionNames: [ 'require', 'import' ],
+	},
```

Comment:
Looks like this has implications for other packages as well :/

Reply:
There's a package patch in https://github.com/WordPress/gutenberg/pull/65250/files?show-viewed-files=true&file-filters%5B%5D=#diff-57cf15f5ef3f76d6b8f1dcfe9ca40e86887fd295f80df3fb2bc0565352520d31 that can be used instead of this change.